### PR TITLE
Wallets for SN instead of keys

### PR DESCRIPTION
**Pull request by EliChin**
For tests we need storage node wallets, so I removed `services/storage/*.key` and placed wallets accordingly.

Signed-off-by: Elizaveta Chichindaeva <elizaveta@nspcc.ru>

### CODE REVIEW

**Review by alexvanin — APPROVED** [review]

File: services/storage/docker-compose.yml [hunk 1]
```diff
@@ -55,7 +57,9 @@ services:
     stop_signal: SIGKILL
     env_file: [ ".env", ".storage.env", ".int_test.env" ]
     environment:
-      - NEOFS_NODE_KEY=/02.key
+      - NEOFS_NODE_WALLET_PATH=./wallet02.json
+      - NEOFS_NODE_WALLET_ADDRESS=NVXXy3hNTvwVEZa2dAibALyJB3Q86aiHvL
```
Comment (realloc) [inline services/storage/docker-compose.yml:61]: Isn't that a redundant option? We only have one address in each wallet. Setting it here may complicate changing key in the future.

Comment (alexvanin) [reply services/storage/docker-compose.yml:61]: As far as I know, neofs-node does not support default wallet addresses.

Comment (fyrchik) [reply services/storage/docker-compose.yml:61]: We should support them, though https://github.com/nspcc-dev/neofs-node/blob/2da8396a9fce37c5df10f9e63e2d6807f77f72c3/cmd/neofs-cli/internal/key/wallet.go#L30

Comment (fyrchik) [reply services/storage/docker-compose.yml:61]: Oh, sorry, this is node, not cli.

Comment (realloc) [reply services/storage/docker-compose.yml:61]: Ok. We need not forget removing this when `neofs-node` gets default address support.

**Review by realloc — APPROVED** [review]

**Pull request merged**